components/dashboard/TextArea/PostButtons.tsx

```diff
@@ -62,18 +63,25 @@ const PostButtons = ({
     );
   }, [notes, selectedNoteId]);
 
-  const saveNoteHandler = () => {
+  const getFromStore1 = async (selectedNoteId: string) => {
```

Comment:
Why is there a `1` in the function name? Any particular reason?

Reply:
That is for in case we have multiple stores. I don't remember why I thought we would need more than 1 though 😅